examples/cndpfwd/l3-fwd.c

```diff
@@ -0,0 +1,117 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) 2022 Intel Corporation.
+ */
+
+#include <cne_fib.h>              // For fib_create, fib_add...
+#include <cne_inet4.h>            // for inet_mtoh64 and inet_h64tom
+#include <net/cne_ether.h>        // for ether_addr
+
+#include "main.h"
+
+#define FIB_RULES_IP   0 /**< JSON index for IP address in a single entry */
+#define FIB_RULES_MAC  1 /**< JSON index for MAC address in a single entry */
+#define FIB_RULES_PORT 2 /**< JSON index for TX port in a single entry */
+
+struct ipv4_l3fwd_fib_rule {
+    uint32_t ip;
+    uint8_t depth;
+    struct ether_addr nh;
+    uint16_t tx_port;
+};
+
+static struct cne_fib *fib;
+
+static int
+l3fwd_fib_populate(struct fwd_info *fwd, struct cne_fib *fib)
+{
+    for (uint16_t i = 0; i < fwd->fib_size; i++) {
+        struct ether_addr mac_le;
+        char *entry[3];
+        char *ip_prefix[4];
+        char *address[2];
+        char *mac_addr[6];
+        int tx_port;
+        uint64_t eaddr = 0;
+
+        /* Parse the comma seperated FIB entry */
+        entry[0] = strtok(fwd->fib_rules[i], ",");
+        for (uint16_t j = 1; j < 3; ++j)
+            entry[j] = strtok(NULL, ",");
+
+        /* Parse the IP address and depth */
+        address[0] = strtok(entry[FIB_RULES_IP], "/");
+        address[1] = strtok(NULL, "/");
+
+        /* Parse the IP address */
+        ip_prefix[0] = strtok(address[0], ".");
+        for (uint16_t j = 1; j < 4; ++j) {
```

Comment:
We do not use {} around single like for, while, if, else, ... lines. Please remove if this is needed.